src/CoreShop2VueStorefrontBundle/Bridge/Model/UrlInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CoreShop2VueStorefrontBundle\Bridge\Model;
+
+interface UrlInterface
```

Comment:
Allow implementators to provide their own slug and path. For example, we're looking to implement exactly that, with multilanguage support.